DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/credit-and-debit-cards/corporate-purchasing-cards/corporate-purchasing-cards.md

## Corporate purchasing cards
Unlike regular credit cards, corporate purchasing cards allow for a much more detailed level of information regarding transactions and drastically reduce the need for paper-based purchase orders and invoice processing. Corporate purchasing card transaction statements contain all the information you would expect in an invoice, such as description of the items in an order, quantity, corresponding tax rates, etc.

> MultiSafepay offers support for integrations with purchasing cards. For more information, email our Integration Team at <integration@multisafepay.com> 

Reviewer: ```suggestion
MultiSafepay supports integrations with purchasing cards. For more information, email the Integration Team at <integration@multisafepay.com> 
```
